app/partials/app.cjsx

```diff
@@ -20,39 +21,36 @@ class GeordiLogger # Make calls to the Geordi API to log user activity
 
   makeGeordi: (projectSlug) ->
     new GeordiClient
-      server: @state?.env
-      projectToken: projectSlug || ''
+      env: @state?.env
+      projectToken: projectSlug || @keys?.projectToken
       zooUserIDGetter: () => @state.user?.id
       subjectGetter: () => @keys?.subjectID
 
   makeHandler: (defType) -> # Once defined, efficiently logs different data to same event type
-    instance = @instance
-    (eventData, eventType) ->
+    (eventData, eventType) =>
         eventType = defType if typeof eventType isnt 'string'
-        if GeordiLogger.tokens.indexOf(instance().projectToken) > -1
-          instance()?.logEvent
-            type: eventType
-            data: "\"#{eventData}\""
+        @logEvent
+          type: eventType
+          data: "\"#{eventData}\""
 
   logEvent: (logEntry) -> # Accepts key/values to make appropriate Geordi logging
-    geordi = @instance()
     newEntry = Object.assign {}, logEntry, @keys
     if GeordiLogger.tokens.indexOf(newEntry.projectToken) > -1
-      @geordi?.logEvent newEntry
-    console.log 'No logger available for event ', JSON.stringify(logEntry) unless @geordi?.logEvent
+      @instance().logEvent newEntry
+      console.log 'No logger available for event ', JSON.stringify(logEntry) unless @instance().logEvent
```

Comment:
I suggest console.warn instead of console.log

Also can we say "No Geordi logger available for event" so it's really clear what we are talking about?